task/testmap.py

```diff
@@ -117,7 +117,6 @@
             'rhel-7-9/live-iso',
             'rhel-7-9/qcow2',
             'rhel-7-9/aws',
-            'rhel-7-9/azure',
```

Comment:
I'd suggest to move it to `_manual` instead, so that it's possible to run `tests-trigger` on it to investigate/fix it.

Reply:
Good point!